### DISCUSSION TITLE
Anomaly Detection in SO 2.4

### DISCUSSION THREAD

**Discussion by brazorfajeje** [Security-Onion-Solutions/securityonion#12494] [Unsupported Versions] [answered]
Has the so-learn project been abandoned in SO 2.4?
Do you plan integrating any machine learning solution in the future?

**Comment by Acewiza** [comment]
I love anomalies!  Variety is the spice of life.  ;-)  

Getting past the human-learning part has always been an issue, for me.

Keep teaching the things - all day long sometimes.  Always doing exactly what they're told, over and over, never changing.

People been working themselves out of jobs since the dawn of mankind.

**Comment by mmartinrami** [comment]
Im interested to. In 2.3  version logscan was available but in 2.4 version was removed.

**Comment by dougburks** [comment, marked as answer]
Logscan was only provided as a BETA feature in 2.3 and never reached full release. 

We are looking to introduce a more integrated ML tool going forward, but there's no ETA yet.
